.gitignore

```diff
@@ -1,6 +1,5 @@
 *.txt 
 *.exe
-.env
```

Comment:
Yep, everything looking good! I think we can add this back after the merge so people don't push their local .env files in case they use sensitive information. Also if anything changes with the installation or usage (like the new .env format) I would make sure to update the readme.